src/components/Slack/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import qs from 'querystring';
+
+import { getSlackToken } from '../../actions/slack';
+import bravoParty from '../../assets/bravo-party.svg';
+import Loader from '../Loader';
+import DisplayCard from '../Cards/DisplayCard';
+import SideNav from '../SideNav';
+
+const Slack = props => {
+  const {
+    location: { search },
+    getSlackToken,
+    slack,
+  } = props;
+  // Parse slack temporary code in URL
+  const parsed = qs.parse(search);
+  const { '?code': code } = parsed;
+
+  useEffect(() => {
+    getSlackToken(code);
+  }, [getSlackToken, code]);
+
+  return (
+    <React.Fragment>
+      <SideNav />
```

Comment:
You can use <> and </> for React.Fragment

Reply:
Okay. Thanks for the fix on the React.Fragment. Working with @samarv on that to fix it on the netlify ENV variables

Reply:
@maximesalomon It has been fixed